### ISSUE TITLE
Implement {{int:}} magic word

### ISSUE THREAD

**Issue by lucaswerkmeister** [wikimedia/jquery.i18n#211]
I just noticed the `{{int:message-key}}` magic word isn’t implemented. Is there a particular reason to omit it or was it just not done yet?

Here’s a simple implementation (using modern JS syntax because I’m lazy):

```js
$.extend( $.i18n.parser.emitter, {
	int( nodes ) {
		return $.i18n( ...nodes );
	},
} );
```

(It doesn’t emit “⧼xyz⧽” for undefined message keys like MediaWiki does, but jquery.i18n doesn’t seem to do that in general, so I’m not sure if its `{{int:}}` should.)

**Comment by ecococox** [comment]
lazım ### #'nın kopyalanması

**Comment by mayankprajapati2324** [comment]
hlo @lucaswerkmeister i want to work on your problem so, please assign me

**Comment by lucaswerkmeister** [comment]
No. This is basically a feature request, so the library’s maintainer(s) should first decide if this ought to be added or not (meanwhile I’ve worked around it as described in the issue description). Also, I’m not a maintainer here, I can’t assign anyone.
